tests/ltp/ext2fs_tests_other_errors.txt

```diff
@@ -188,7 +187,6 @@
 /ltp/testcases/kernel/syscalls/flock/flock03
 /ltp/testcases/kernel/syscalls/flock/flock04
 /ltp/testcases/kernel/syscalls/flock/flock06
-/ltp/testcases/kernel/syscalls/fork/fork01
```

Comment:
Is this test now passing? Is there anything to update in readme?

Reply:
Hey, it’s marked as a flaky test so it is removed from the failing list